WA3CRUD/WA3CRUD/Controllers/MarketController.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using WA3CRUD.Models;
+
+namespace WA3CRUD.Controllers
+{
+    [Route("api")]
+    [ApiController]
+    public class MarketController : ControllerBase
+    {
+        static readonly IMarketable phonemarket = new Market();
+
+
+        [HttpGet("Products")]
+        public IEnumerable<Product> GetAllProducts()
+        {
+            return phonemarket.GetAll();
+        }
+
+
+        [HttpGet("Products/id")]
```

Comment:
1. all the parameters in route we put in {}
2. do not use Capital letters in route
3. always put parameter constraints (":int") for the route parameters if it is not string parameter  

this route should be "products/{id:int}"